presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -763,6 +764,35 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
         return TableStatistics.empty();
     }
 
+    @Override
+    public void createSchema(JdbcIdentity identity, String schemaName)
+    {
+        String sql = "CREATE SCHEMA " + quoted(schemaName);
```

Comment:
inline

Reply:
Some datatabase could have special syntax for that. Possibly we could have a dedicated method to to generate drop and create schema SQLs that implementation of JDBC based connector could override.

Reply:
`createSchema` is so simple that `BaseJdbcClient` subclass can override the whole method, if needed.